force-app/main/default/classes/LookUpController.cls

```diff
@@ -0,0 +1,29 @@
+public with sharing class LookUpController {
+
+    @AuraEnabled(Cacheable = true)
+    public static List<SObject> getRecordList(String addInfo, String sObjectName, String searchKey){
```

Comment:
@TsSlava 
1) rename "addInfo" and add ability to add more than one field. Also make it secured.
2) getRecordList rename to getRecords